cfme/tests/physical_infrastructure/ui/test_physical_server_details.py

```diff
@@ -52,6 +52,28 @@ def test_physical_server_details_dropdowns(physical_server):
     assert "Timelines" in monitoring_items
 
 
+def test_network_devices(physical_server):
+    """Navigate to the Network Devices page and verify that the page is displayed"""
+
+    if physical_server.num_network_devices() != "0":
```

Comment:
Optional: more compact variant to remove steps duplication
devices_num = physical_server.num_network_devices()
network_device_view = navigate_to(physical_server, 'NetworkDevices')
assert (
    network_device_view.is_displayed if devices_num !='0' else not network_device_view.is_displayed
)